tools/constructor-test-helper.js

```diff
@@ -1,7 +1 @@
-import { expect } from '@open-wc/testing';
-
-export function runConstructor(nodeName) {
-	const ctor = customElements.get(nodeName);
-	expect(ctor).to.not.be.undefined;
-	document.createElement(nodeName);
-}
+export { runConstructor } from '@brightspace-ui/testing';
```

Comment:
Hmm, I think I have to make `@brightspace-ui/testing` a non-dev dependency if we want to do this actually. Thoughts? The other option is to leave the copy here.

Reply:
Hmm, I don't really want it in BSI... I think I'd vote to just keep the copy

Reply:
Yeah agreed. Ok I've restored the copy -- technically there's was a dependency on `@open-wc/testing` in the same way... but chances are consumers have it already.

Reply:
Was just gonna ask how that was working lol, but right, they'll have that. But likely not our new library yet 👍